.github/workflows/helm_charts_scalar.yml

```diff
@@ -12,27 +12,27 @@ on:
   workflow_dispatch:
 
 env:
-  HELM_VERSION: v3.5.2
+  HELM_VERSION: v3.11.3
```

Comment:
Based on the Helm document, I chose helm v3.11.x which is the latest supported version.
https://helm.sh/docs/topics/version_skew/